conf/set_config.go

```diff
@@ -186,6 +246,22 @@ func WriteGlobalConfig(ng Global) {
 	fs.Replace(gcPath, newMarhsalled)
 }
 
+// WriteGlobalProjectSettings takes an updated Project object and merges it with all of the
+// Projects defined in the global config, finally saving the whole global config back to disk
+func WriteGlobalProjectSettings(p *Project) error {
+	g := GetConfig().Global
+
+	for k, v := range g.Projects {
+		if v.Name == p.Name {
+			g.Projects[k] = *p
+		}
+	}
+
+	WriteGlobalConfig(g)
```

Comment:
No check on `err` here? If this function can never return an error, probably better to remove the return type

Reply:
good find - will fix !